static/index.html

```diff
@@ -3,15 +3,19 @@
     <head>
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Ajax form - Go backend</title>
         <link rel="stylesheet" href="./styles/style.css">
-       <!-- <script src='//ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js'></script> -->
-       <script src="./scripts/jquery-3.4.1.min.js"></script>
+       <!-- <script src='//ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js'></script>-->
+        <!-- Yes. For personal projects it is best to pull down
+         the version of jQuery you want directly into your project.
+          Also version 1.n.n of jQuery is way too old -->
+       <script src="scripts/jquery-3.4.1.min.js"></script>
     </head>
     <body>
-        <form action="" method="post" id="comment-form">
-            <label for="name">Your Name:</label>
+        <form action="#" method="post" id="comment-form">
+            <label for="username">Your Name:</label>
```

Comment:
The `for` of the label should match the `name` and / or `id` of the corresponding `input` or `textarea`